vite.config.js

```diff
@@ -0,0 +1,23 @@
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+
+import pkg from './package.json';
+
+export default defineConfig({
+    build: {
+        lib: {
+            // Could also be a dictionary or array of multiple entry points
+            entry: resolve(__dirname, 'src/client.js'),
```

Comment:
We used to provide 2 libraries: one for the client and one for the server. I only see the client one here.